src/main/java/org/folio/circulation/services/agedtolost/ChargeLostFeesWhenAgedToLostService.java

```diff
@@ -116,7 +116,9 @@ public ChargeLostFeesWhenAgedToLostService(Clients clients) {
   }
 
   private CompletableFuture<Result<Loan>> updateLoanBillingInfo(LoanToChargeFees loanToChargeFees) {
-    final Loan updatedLoan = loanToChargeFees.getLoan().setLostItemHasBeenBilled();
+    final Loan updatedLoan = loanToChargeFees.getLoan()
+      .setLostItemHasBeenBilled()
+      .useNoAction();
```

Comment:
Should setting that a lost item has been billed always have no action? If so, could this been done in that method?

Reply:
No, in some cases we should mark item as lost and paid and set is billed.

Reply:
And that should have an action?

Reply:
Yes, the action is `closedLoan` in this case.